ultralytics/engine/exporter.py

```diff
@@ -878,11 +886,12 @@ def export_saved_model(self, prefix=colorstr("TensorFlow SavedModel:")):
         onnx2tf.convert(
             input_onnx_file_path=f_onnx,
             output_folder_path=str(f),
-            not_use_onnxsim=True,
+            not_use_onnxsim=not self.end2end,  # use onnx simplify for end2end models
```

Comment:
YOLOv10 won't work with `onnxslim` so using the `onnxsim` feature built into `onnx2tf` instead